main_app/models.py

```diff
@@ -46,3 +46,64 @@ class UserModel(PermissionsMixin, AbstractBaseUser):
 
     def __str__(self):
         return self.email
+
+
+class MustModel(models.Model):
+    game_id = models.IntegerField()
+    user = models.ForeignKey(UserModel, on_delete=models.CASCADE)
+    is_deleted = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f"{self.user.username} {self.game_id}"
+
+
+class GameModel(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=100)
+    rating = models.DecimalField(decimal_places=0, max_digits=3, null=True)
+    version_title = models.CharField(max_length=100, null=True)
+    aggregated_rating = models.DecimalField(decimal_places=0, max_digits=3, null=True)
+    summary = models.CharField(max_length=2500, null=True)
+    first_release_date = models.DateField(null=True)
+    rating_count = models.DecimalField(decimal_places=0, max_digits=3, null=True)
+    aggregated_rating_count = models.DecimalField(decimal_places=0, max_digits=3, null=True)
+
+
+class GenreModel(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=50)
```

Comment:
Many genres has many games. It's many to many relationship. The correct way to define model here is to:
```
class Genre(models.model):
    name = models.CharField(max_length=100, unique=True)
    games = models.ManyToManyField(Game, related_name="genres")
```
https://docs.djangoproject.com/en/2.2/topics/db/examples/many_to_many/
https://habr.com/ru/post/193380/ (Part about M2M)

`GenreGameModel` is not needed